### DISCUSSION TITLE
[小说]咖啡馆恐怖小说第二章，要看第一章的可以点我主页或者往下找找，封面跟这个一样

### DISCUSSION THREAD

**Discussion by aaa00073** [share121/inter-knot#497] [其他]
第二章 逃跑
    陈往喝了九个小时咖啡了，这一杯也没见底。

　　他也没感觉到撑。

　　但是总感觉这这咖啡口感有点粘稠，

　　还有股腥咸味和淡淡铁味。

　　但是再奇怪，陈往也只有硬着头皮喝。

　　他不知道如果眼前的女人发现他看出了异常，会发生什么，他也不敢赌。

　　为什么会变成这个样子啊！

　　明明只是一个普普通通的高中生！

　　只是想出门喝杯咖啡而已，为什么自己会走到这里来啊！

　　早知道会这样，陈往一定不会听他同学的，去找咖啡店尝试什么手磨咖啡，在家喝速溶不是挺好的吗？

　　但是已经没办法后悔了。

　　事已至此。

　　陈往一口咖啡灌进嘴里。

　　呕！

　　陈往感到一阵恶心和反胃。

　　但是他强行忍住，把咖啡咽进了喉咙，努力使脸上的表情平静，说道：

　　“继续，刚才讲到哪里了？”

　　陈往说的时候，眼睛还是移不开女人那已经错位了的脖颈。

　　错位还在变大。

　　照这个速度，最多还有半个小时，就会掉下来。

　　不，别！

　　那该不会是什么倒计时吧？

　　等到那一刻到来，女人就会真正亮出她的獠牙，把他杀死？

　　可是现在这种煎熬……

　　女人讲故事会总是停顿，

　　此时，必须要陈往喝一口咖啡，回复一句话，才会继续。

　　少一步都不行！

　　可是陈往很确定，再喝一口的话，

　　他一定会呕吐出来！

　　陈往视线上移，看向女人的脸。

　　经过苍白色的嘴唇。活人的嘴唇是这个颜色吗？

　　沿着高挺的鼻梁向上。到底有没有在呼吸，难说。

　　一直到女人的双眼。

　　女人的眼睛从未看向过这边，每过一会儿，眼球就会挤开眼皮突出来，然后女人就会把它按回去。

　　她又开始讲述了。

　　“我来到了一家咖啡店，老板说，这家店只有晚上开。”

　　“我说，我是来应聘的。”

　　“老板说，活人不可以在这里工作。”

　　“我问老板，晚上的客人也不是活人吗？”

　　“老板说，是活人，但……”

　　女人又停顿了。

　　陈往端起咖啡：“但什么？”

　　女人放下手，撑在桌面，身体前倾说道：

　　“但是每一位客人，都会摆上下一位客人的餐桌。”

　　“所以，我算不算上一位客人？”

　　女人说完这句话，陈往忽然察觉到不对了。

　　有变化！

　　这次跟之前每次重复这个故事的版本都不一样！

　　怎么回事？

　　每一位客人，都会摆上下一位客人的餐桌……

　　那自己正在喝的是什么！？

　　陈往此时忽然感觉嘴里味道有点奇怪。

　　他放下正在喝的咖啡杯一看，

　　呵，哪还有什么咖啡啊，

　　杯子里完全变成了一片血红！

　　“呕——”

　　陈往再也难以忍受地捂住了嘴，他终于知道“咖啡”那奇怪的口感是为什么了。

　　他一手撑着桌子，头转向一边，剧烈地呕吐起来。

　　血液不断从他口中吐出，陈往只感觉之种发自内心的恶心！

　　吐了很久，陈往才抬起头来。

　　目光正好对上了玻璃反光中的自己。

　　嘴角附近全是鲜红色，牙齿缝里还是是血红色！

　　双眼也布满血丝，大概是因为神经紧绷太久。

　　将目光收回。

　　陈往刚刚重新转向女人，还没抬头，他就感觉后脊一僵。

　　等等！

　　刚才看到了什么？

　　陈往确定自己看到了异常的东西。

　　但是，他不敢把目光转回去寻找。

　　只扫过一眼，只有些许印象。

　　是玻璃中的反光！

　　陈往想起来了，是反光！玻璃根本没有映射出这个女人的身影！

　　而是……

　　陈往自己！？

　　砰——

　　一声闷响，女人的脑袋重重砸在了她面前的盘子上。

　　眼球弹了出来，落到了陈往盘子里。

　　“……”陈往。

　　此刻，陈往手里握的最紧的，就是那把金属的勺子。

　　这些微的寸铁，能带给他聊胜于无的底气。

　　在瞳孔剧烈震颤的陈往注视下，

　　那盘中的脑袋开口了……

　　“客人，请用餐。”

　　“祝您……用餐愉快。”

　　哐当一声，陈往再也坐不住了，他慌乱的站起来，碰倒了身后的椅子。

　　咖啡杯被打翻，血液流满整张桌子，随后又流到地上。

　　陈往哪还敢停留。

　　终于结束了那种煎熬，陈往拖着酸麻的腿就想跑。

　　整座咖啡馆漆黑一片。

　　只有陈往刚才坐的位置，有些许亮光。

　　逃跑中的陈往回头看去。

　　那一小片光正在逐渐远离。

　　坐在桌边的女人并没有追上来，也没有做任何动作。

　　他长呼一口气，不敢停下脚步。

　　渐渐的，一切都变成了一片黑暗。

　　什么也看不见。

　　陈往只能听见自己的喘息声。

　　一刻不停地跑了许久。

　　前面似乎有亮光？

　　终于能离开这里了吗？

　　陈往内心一阵欣喜，不由得加快了脚步。

　　然而，随着微光的靠近，陈往的表情却逐渐凝固。

　　那是一扇门。

　　深夜咖啡馆的店门……

　　“为什么？”

　　“为什么又回到这里了？”

　　陈往感受到一种无力感。

　　从店外的玻璃看进去，陈往甚至能看到那张有光亮的桌子，倒在地上的椅子。

　　以及……那个女人。

　　陈往走到玻璃面前。

　　他的拳头逐渐捏紧。

　　恐惧，正逐渐转化为愤怒。

　　咚——

　　陈往一拳砸在玻璃上，发出一声沉闷的声响。

　　伴随着声音响起的瞬间，

　　店内的女人，突然消失了。

　　陈往刚反应过来自己做了什么，他后退好几步。

　　转身就跑。

　　这一次，陈往使出了自己的全力，从未有过一次跑得这样快。

　　奔跑着，陈往慢慢感受到有雨落在脸上。

　　雨越来越多了。

　　前方，是一个光点。

　　逐渐放大，是一个一人宽的缝隙。

　　但是透过缝隙，陈往看到了马路！

　　看到了车辆，霓虹灯，打伞的路人……

　　用尽全力，陈往从缝隙中冲了出去。

　　逃出来了！

　　竟然逃出来了！

　　回头看去，那里根本没有什么缝隙，只有一堵墙。

　　陈往从地上爬起来，不顾路人异样的目光，张开双臂，迎接着漫天的大雨。

　　活着的滋味，从未如此真实！

　　陈往感受着自由的呼吸。

　　怎么感觉脖子有点痒……

　　陈往摸了摸脖颈处，确实有种奇怪的感觉，就是觉得痒丝丝的。

　　微微用力，

　　一条缝被翻开了，几滴液体流出。

　　然后，陈往整个脑袋开始错位，向一侧滑去……

　　呵，

　　完蛋。
![IMG_20240724_220546](https://github.com/user-attachments/assets/0de8cbd7-a418-47f9-93b5-b5621e89280f)

**Comment by swyh51982** [comment]
求更新，大家什么都会做的
